docs/streams/developer-guide/config-streams.html

```diff
@@ -667,12 +668,14 @@ <h4><a class="toc-backref" href="#id30">probing.rebalance.interval.ms</a><a clas
           <span id="streams-developer-guide-processing-guarantee"></span><h4><a class="toc-backref" href="#id25">processing.guarantee</a><a class="headerlink" href="#processing-guarantee" title="Permalink to this headline"></a></h4>
           <blockquote>
             <div>The processing guarantee that should be used.
-              Possible values are <code class="docutils literal"><span class="pre">"at_least_once"</span></code> (default),
-              <code class="docutils literal"><span class="pre">"exactly_once"</span></code> (for EOS version 1),
-              and <code class="docutils literal"><span class="pre">"exactly_once_beta"</span></code> (for EOS version 2).
-              Using <code class="docutils literal"><span class="pre">"exactly_once"</span></code> requires broker
-              version 0.11.0 or newer, while using <code class="docutils literal"><span class="pre">"exactly_once_beta"</span></code>
-              requires broker version 2.5 or newer.
+              Possible values are <code class="docutils literal"><span class="pre">"at_least_once"</span></code> (default)
+              and <code class="docutils literal"><span class="pre">"exactly_once_v2"</span></code> (for EOS version 2).
+              Deprecated config options are <code class="docutils literal"><span class="pre">"exactly_once"</span></code> (for EOS alpha),
```

Comment:
Do we need to elaborate on all deprecated configs?

Reply:
I think given we have just rolled this out, and many users are likely still in the process of upgrading their brokers to a sufficient version, we should continue to mention EOS alpha. And then we may as well mention beta as well for consistency. Don't feel too strongly though